### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by vermasavita**
features:
1.  add add to cart & add to wishlist
2. Add go to cart with link of cart page

style:
1. Add wishlist.css
2. changes some minor styles

Add link to website logo

https://user-images.githubusercontent.com/82254618/162210094-0518d09d-2b9a-4166-bd2b-59f1ecc57f36.mov

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
* Code Looks Good
* Check your build issue

**Review by pradipgharge — APPROVED** [review]
Code looks good , check your build issue

**Comment by pnchinmay** [comment]
UI looks good
Left a few sanity comments

File: src/components/card/card.jsx [hunk 1]
```diff
@@ -28,7 +59,48 @@ const ProductCart = ({
...
+                productPrice,
+                productRating,
+              },
+            })
+          }
+        >
+          Add to Cart
+        </button>
+      }
+      {/* <button
```
[the remark this answers is not in the document]
Comment (vermasavita) [reply src/components/card/card.jsx]: Done

File: src/components/nav/navbar.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 :root {
     --ecomm-primary-color: rgba(15, 141, 173, 0.9);
     --secondary-color: rgb(255, 255, 255);
-    --head-text-color: rgb(116, 115, 115);
+    --head-text-color: rgb(195, 203, 205);
```
[the remark this answers is not in the document]
Comment (vermasavita) [reply src/components/nav/navbar.css:4]: Noted

**Pull request merged**